data/privacy_notices/privacy_notice_templates.yml

```diff
@@ -0,0 +1,227 @@
+privacy_notices:
+- name: Essential
+  description: This website uses cookies and other methods which are essential for you to use the website and its features.
+  internal_description: These cookies and services are essential for you to browse the website and use its features, such as accessing secure areas of the site. Cookies that allow web shops to hold your items in your cart while you are shopping online are an example of strictly necessary cookies. These cookies will generally be first-party session cookies. While it is not required to obtain consent for these cookies, what they do and why they are necessary should be explained to the user.
+  regions:
+    - eu_be
+    - eu_bg
+    - eu_cz
+    - eu_dk
+    - eu_de
+    - eu_ee
+    - eu_ie
+    - eu_el
+    - eu_es
+    - eu_fr
+    - eu_hr
+    - eu_it
+    - eu_cy
+    - eu_lv
+    - eu_lt
+    - eu_lu
+    - eu_hu
+    - eu_mt
+    - eu_nl
+    - eu_at
+    - eu_pl
+    - eu_pt
+    - eu_ro
+    - eu_si
+    - eu_sk
+    - eu_fi
+    - eu_se
```

Comment:
Currently listing all the EU regions out individually, let's double check this is how we want to define this

Reply:
@mfbrown